tests/network/vmi_networking.go

```diff
@@ -57,6 +57,13 @@ import (
 	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
+const (
+	vmiNetworkingEchoCmd = "echo $?\n"
```

Comment:
This could have a more generic name, and maybe even be moved to the test utils.
Maybe something like "echoLastReturnValue"?